lstchain/reco/r0_to_dl1.py

```diff
@@ -167,6 +168,7 @@ def get_dl1(
     telescope_id,
     dl1_container=None,
     custom_config={},
+    lhfit_args={}
```

Comment:
Don't use mutable default arguments, use None.

Reply:
@maxnoe Should I also solve this for "custom_config"? From what I see it implies adding a check for None in at least "replace_config" or a if custom_config is None: custom_config={} at the start of the get_dl1 function.